src/PieChart/PieArcSeries/PieArcLabel.tsx

```diff
@@ -67,20 +84,39 @@ export class PieArcLabel extends PureComponent<PieArcLabelProps> {
           duration: 0.1
         }}
       >
-        <title>{text}</title>
-        <text
-          dy={padding}
-          fill={fontFill}
-          fontSize={fontSize}
-          fontFamily={fontFamily}
-          textAnchor={textAnchor}
-          style={{
-            shapeRendering: 'crispEdges',
-            transform: `translate3d(${posX}px,${posY}px, 0)`
-          }}
-        >
-          {text}
-        </text>
+        {typeof text === 'string' ? (
+          <>
+            <title>{text}</title>
```

Comment:
The title feels odd at this element placement.

Reply:
I don't know. It was there before. I've thought this is something for screen readers, because it does not renders at all. Should I remove it?